### ISSUE TITLE
[Token] Color semantic tokens update (Oct, 8)

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/ouds-ios#177]
### Description

Color semantic tokens have been updated on October, 8.

⚠ As usual, the files generated by the parser are not perfect, so please adapt to your current tokens files.

```diff
public enum ColorSemanticTokens {
+    public static let colorContentTransparentDefault = colorTransparentBlack0
    public static let colorBrandWarningLowest = colorFunctionalSun100
    public static let colorBrandWarningHighest = colorFunctionalSun900
    public static let colorBrandWarningHigh = colorFunctionalSun600
    // ...
    public static let colorBrandAttractiveDefault = colorFunctionalSun500
+    public static let colorBorderTransprentDefault = colorTransparentBlack0
+    public static let colorBgTransparentDefault = colorTransparentBlack0
    public static let colorDecorativeSkinTint900 = colorDecorativeDeepPeach900
```

⚠ There's a typo in `colorBorderTransprentDefault` as you can see, a missing "a" in the word "transparent". You can directly use the right naming: `colorBorderTransparentDefault`.

**Comment by pylapp** [comment]
@julien-deramond  Raw tokens are missing :(

**Comment by pylapp** [comment]
ℹ️ Released with version [v0.4.0](https://github.com/Orange-OpenSource/ouds-ios/releases/tag/0.4.0) 🚀
